enterprise/backend/test/metabase_enterprise/sso/integrations/sso_utils_test.clj

```diff
@@ -10,7 +10,7 @@
        "/"
        "/test"
        "localhost"
-       "localhost:3000"
+       ;; "localhost:3000" ; URI thinks `localhost` here is scheme
```

Comment:
should this be explicitly an invalid URL then? (i.e. move it to the other testing block?)